API.md

```diff
@@ -0,0 +1,204 @@
+# API docs
+
+The rails server exposes a JSON API. All API endpoints are prefixed by `/api/v1`.
+
+## Locations
+
+### `GET /locations/countries`
+
+Lists all locations that are countries
+
+#### Query parameters
+
+* `topojson` - if present, the location's topojson structure will be sent with the object.
```

Comment:
Should this be treated as a boolean? i.e.: `topojson=true`? Or it doesn't matter?

Reply:
It doesn't matter. The only thing required is that the parameter is present.